data/engine-cli/docker_service_update.yaml

```diff
@@ -440,7 +442,7 @@ options:
       description: Limit maximum number of processes (default 0 = unlimited)
       deprecated: false
       hidden: false
-      min_api_version: "1.41"
+      min_api_version: "1.52"
```

Comment:
Interesting; curious if this was some mistake somewhere; let me check

Reply:
Nope; it is a bug / mistake; looks like some copy/pasta same flag-name repeated here; https://github.com/docker/cli/blob/93fa57bbcd08f2f5be7f6cf22f4273a2b5a49e71/cli/command/service/opts.go#L905-L910